gcn/notices/km3net/alert.schema.json

```
@@ -0,0 +1,85 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/km3net/alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "unevaluatedProperties": false,
+  "title": "KM3NeT Alerts",
+  "description": "Candidate neutrino observation reported from the KM3NeT online analysis pipelines.",
+  "allOf": [
+    {
+      "$ref": "../neutrino/Alert.schema.json",
+      "description": "Neutrino alert schema. Alert info and most probable Localization (Time, Position, Error and Skymap if available)."
+    },
+    {
+      "$ref": "../core/AdditionalInfo.schema.json",
+      "description": "Nb. of pipeline triggering on this event, targeted FAR and details on selection."
+    }
+  ],
+  "properties": {
+    "$schema": true,
+    "pipeline": {
+      "enum": [
+        "exceptional_evt_arca",
+        "exceptional_evt_orca",
+        "multiplet_arca",
+        "multiplet_orca"
+      ],
+      "description": "Pipeline issuing this notice: either single, exceptional (high-energy), event or multiple coincident events, from KM3NeT/ARCA or KM3NeT/ORCA data."
```

Single isn't in the enum, unless it's 
```suggestion
      "description": "Pipeline issuing this notice: either a single exceptional (high-energy) event or multiple coincident events from KM3NeT/ARCA or KM3NeT/ORCA data."
```

---

Also since you list ORCA or ARCA in the instrument, do you also need to specify here?